monkestation/code/modules/bloodsuckers/structures/bloodsucker_crypt.dm

```diff
@@ -282,6 +284,9 @@
 			balloon_alert(user, "someone else's vassal!")
 			return FALSE
 
+	if(isanimal_or_basicmob(target))
```

Comment:
Would this be better as a simple !ishuman

Reply:
They'd effectively do the same thing, but JUST to be safe, I'll change that!